README.md

```diff
@@ -91,27 +102,15 @@ $ source setup
 >
 > Here is a workaround to fix the build issue [link](https://lists.yoctoproject.org/g/docs/topic/yocto_workaround_for/106220010) \
 
-7. Build the image by using the provided `Makefile`. 
+
+7. Build the images!
+
+Your host is now ready to build images.  
+
 > **Note:** Make sure you are exporting the env vars you want to enable/disable yocto build time configuration. \
 > Please inspect the [env_files](./env_files/) and export the desired ones as env variables before trigger the build. \
 > e.g. export DEBUG_TWEAKS_ENABLED=1. This builds the image in debug mode and enable root ssh access for dev/debug purposes
 
-```
-$ make build
-```
-
-8. Generate the measurements values after building the image. They will be located in measurements directory
-```
-$ make gen-measurements
-```
-> **Note:** to generate the measurements, you need to make sure that you have `python3`, `libssl-dev` and the `signify` module installed.
```

Comment:
I'll bring this back some time later

Reply:
why delete it in the first place? it is still valid because that target is coming from the yocto-scripts